lib/bn.js

```diff
@@ -56,7 +56,7 @@
 
   BN.isBN = function isBN (num) {
     return num !== null && typeof num === 'object' &&
-      num.constructor.name === 'BN' && Array.isArray(num.words);
+      num.constructor.name === BN.name && Array.isArray(num.words);
```

Comment:
If we wanted to check against just one instance of `BN` - we would use `obj instance of BN`, right? :wink: